packages/slice-machine/src/utils/tracker.ts

```diff
@@ -38,6 +43,7 @@ export class ClientTracker {
     eventType: EventType,
     attributes: Record<string, unknown> = {}
   ): void {
+    if (this.tracking === false) return void 0;
```

Comment:
!tracking

Reply:
return alone also

Reply:
can do, there's a few extra warnings in this one :/